### PR TITLE
Fix for project versions dropdown

### PR DESCRIPTION
**Pull request by walterdeboer**
### Description

Fixed project versions dropdown by including a list of versions in the API call result.

image

### Addressed Issue

Fixes: #397

### Additional Details

Also added metrics in the API call result and updated UI, to minimize round trips.

Updated some dependencies to get rid of some sonatype-lift warnings, ignored the rest 

Needs: https://github.com/DependencyTrack/dependency-track/pull/2581

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#pull-requests)
- [ ] This PR introduces new or alters existing behavior, and I have updated the [documentation](https://github.com/DependencyTrack/dependency-track/tree/master/docs/_docs) accordingly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +33,7 @@
     "core-js": "2.6.12",
```
[the remark this answers is not in the document]
Comment (walterdeboer) [reply package.json]: @sonatype-lift ignore

**Comment by valentijnscholten** [comment]
What will it look like with 63 versions?

I think the screenshot is from the cpan PR 🙂

**Comment by walterdeboer** [comment]
> What will it look like with 63 versions?
> 
> I think the screenshot is from the cpan PR 🙂

I would expect CoreUI to handle the 63 versions nicely, but it looks like it doesn't. Hope this is already fixed in future versions of core-ui. (#445) 

Updated the screenshot.

File: package.json [hunk 2]
```diff
@@ -59,6 +61,7 @@
```
[the remark this answers is not in the document]
Comment (walterdeboer) [reply package.json]: @sonatype-lift ignore

File: package.json [hunk 3]
```diff
@@ -18,64 +18,60 @@
...
+    "@coreui/coreui": "^2.1.16",
+    "@coreui/coreui-plugin-chartjs-custom-tooltips": "^1.3.1",
+    "@coreui/icons": "^0.4.1",
+    "@coreui/vue": "^2.1.2",
+    "@johmun/vue-tags-input": "^2.1.0",
+    "@vue/babel-helper-vue-jsx-merge-props": "^1.4.0",
+    "@vue/babel-preset-jsx": "^1.4.0",
+    "axios": "^0.27.2",
+    "bootstrap": "^4.6.2",
+    "bootstrap-table": "^1.21.3",
```
[the remark this answers is not in the document]
Comment (walterdeboer) [reply package.json:30]: @sonatype-lift ignore

**Comment by walterdeboer** [comment]
> What will it look like with 63 versions?

#449 will show them correctly

**Comment by nscuro** [comment]
@walterdeboer With the API server PR merged (at last!), could you please resolve the merge conflicts in this PR as well?

**Pull request merged**